lebedigital/demonstrator_scripts/beam_design.py

```diff
@@ -150,15 +153,16 @@ def beam_section_design(
             S= min(0.25 * height, 200, slmax)
     #output final design
     out = {}
-    out["top_steel_dia[mm]"] = steel_dia
+    out["top_steel_dia[mm]"] = steel_dia * ureg('mm')
```

Comment:
isn't the steel_dia by using the decorator automatically a length unit? Same with S and As1 (based on the computations using units? So instead of adding them at the end, could we just make all the computations with units?

Reply:
No, using the wrapper, only checks the input and converts it if necessary.
Within the function there are no units.
You can prescribe output (c.f. youngs_modulus_approximation), but as this is a dictionary, this can not be done using the wrapper decorator.